cogs/utilities.py

```diff
@@ -20,6 +20,34 @@ class Utilities(commands.Cog):
     General Utilities
     """
 
+    @commands.command()
+    @commands.has_any_role(*adminRoles)
+    @commands.bot_has_permissions(manage_messages=True, manage_roles=True)
+    async def checkRoles(self, ctx: Context):
+        # check server wide perms
+        perms = None
+        for r in ctx.me.roles:
+            print(r, ctx.me.name)
+            if r.name[0:4] == 'Demo': # need better way to do this crap.
+                perms = r.permissions
+                break
+        # does't work.. perms = ctx.guild.permissions_for(ctx.me)
+        if perms and not perms.move_members:
+            await ctx.send('Bot requires Move Members permission(s)')
+        else:
+            await ctx.send(
+                """
+If you are reading this demoBot has all the permissions it needs for: `nextV2`.
+                
+Just double check the help voice and text channels have matching names: 
+e.g. `help-1` `help-1` and not `help 1` `help-1`.
+                
+Optionally:
+You can hide the help channels for @everyone and @verified.
```

Comment:
Maybe put @everyone and @verified in `so that it appears as code?` instead of mentioning all the channels

Reply:
no cause that will tag everyone with the role which is not what is wanted.

Reply:
need to escape it